services/src/unit-test/java/uk/gov/moj/sdt/services/BulkSubmissionServiceTest.java

```diff
@@ -278,9 +247,9 @@ public void testSubmissionWithMultipleRequests() throws IOException {
         bulkSubmissionService.saveBulkSubmission(bulkSubmission);
 
         // Verify the Mock
-        EasyMock.verify(mockGenericDao);
+        verify(mockGenericDao, times(2)).persist(bulkSubmission);
```

Comment:
I'm probably not following the code correctly but why is this checking that persist() is invoked twice? This method only sets up mockGenericDao to invoke it once.

Reply:
removed it, not needed

Reply:
Confirmed removal of persist.